packages/kit/src/exports/index.js

```diff
@@ -110,6 +110,18 @@ export function redirect(status, location) {
 	);
 }
 
+export const acceptResponse = new Response(null, {
+	status: 200
+});
+
+/**
+ * Accepts a websocket upgrade request. When called during request handling, SvelteKit will accept the websocket upgrade request.
+ * @return {Response} This response instructs SvelteKit to accept the websocket upgrade request.
```

Comment:
It seems to be referred to as "WebSocket" everywhere except the upgrade header value which is "websocket" in lowercase.
```suggestion
 * Accepts a WebSocket upgrade request. When called during request handling, SvelteKit will accept the WebSocket upgrade request.
 * @return {Response} This response instructs SvelteKit to accept the WebSocket upgrade request.
```